External/Plugins/ASCompletion/Model/PathModel.cs

```diff
@@ -525,6 +517,20 @@ public bool HasFile(string fileName)
             }
         }
 
+        public bool TryGetFile(string fileName, out FileModel value)
+        {
+            if (!IsValid)
```

Comment:
Should it in the locked section?

Reply:
it's strange moment

Reply:
The rest of code doesn't include the IsValid section inside the lock, so I left the same logic.